doc/nrf/ug_wifi.rst

```diff
@@ -0,0 +1,147 @@
+This adds redundancy which can be used at the receiver in an attempt to recover the original data content.
+
+Multiple schemes (and rates) are available, depending on the version of the standard in use and the channel conditions.
+Acknowledgements (ACKs) are used for unicast packet transmissions, with every successful packet reception resulting in an ACK transmission back to the originator confirming reception.
+
+The RF medium also necessitates security to achieve privacy of the over-the-air packet transmissions.
+Again, multiple schemes are available depending on the version of the standard in use, though over time the older schemes tend to be deprecated as vulnerabilities emerge.
+The schemes are referred to as Wi-Fi Protected Access (WPA), with WPA3 being the latest version (mandated for all new WFA certifications).
+
+Wi-Fi Protocol stack
+********************
+
+The IEEE802.11 specifications only define the PHY and MAC layers of the Wi-Fi protocol, with the remaining upper layers being common with other IP-based technologies, in particular Ethernet (IEEE802.3).
+
+The following diagram details the full Wi-Fi protocol stack.
+
+
+.. figure:: /images/wifi_protocol_stack.svg
+   :alt: Wi-Fi protocol stack
+
+   Wi-Fi protocol stack
+
+Network stack
+=============
+
+At the top of the stack are application orientated network protocols, for example Matter, MQTT and LwM2M.
+These are agnostic with respect to the underlying communication protocols.
+The next level down is the TCP/IP stack, common to all IP-based communication protocols, such as Wi-Fi, Ethernet, and Thread.
+
+Wi-Fi host stack
+================
+
+Under the common network stack is the first Wi-Fi centric layer, the Wi-Fi host stack.
+There are two logical parts to the Wi-Fi host stack, namely the Wi-Fi driver and the supplicant.
+
+Wi-Fi driver
+------------
+The Wi-Fi driver is responsible for presenting APIs for interfacing with the network stack and the supplicant, as well as handling communication with the Wi-Fi baseband, including managing any physical transport layers, for example SPI and USB.
+
+Supplicant
+----------
+The supplicant is primarily responsible for implementing the authentication phase of WPA (WPA2/WPA3).
+Authentication deals with security before association, and is typically divided into two categories of algorithm, Personal and Enterprise.
+Personal authentication uses the pre-shared key (PSK) approach, which to the end user is usually presented as a network passphrase.
+Enterprise security uses the 802.1X authentication protocol, and typically involves user certificates and a centralized authentication server, for example RADIUS.
+Apart from WPA authentication, the supplicant performs other duties in Wi-Fi associated with joining a network such as network scanning, key exchange, and roaming support.
+The supplicant used with nRF70 Series devices is a Zephyr port of `WPA Supplicant`_.
+
+MAC layer
+=========
```

Comment:
I want to make the text heading levels follow the diagram more closely, in particular have the major headings aligned with the black column on the rhs. Then we can lose some of the text such as "continuing down the stack", since it will be easier to cross reference with the diagram.

```suggestion
Wi-Fi baseband
=========

The Wi-Fi baseband consists of the bottom two layers of the protocol stack, namely the Medium Access Control (MAC) layer and the Physical (PHY) layer. These two layers are fully specified in the IEEE802.11 standard.

MAC layer
---------
```